addon/components/-private/simple-checkbox.js

```diff
@@ -11,6 +11,8 @@ export default Component.extend({
     'indeterminate',
     'type',
     'value',
+    'dataTestSelectRow:data-test-select-row',
+    'dataTestCollapseRow:data-test-collapse-row',
```

Comment:
Similarly here, are these just static attributes we want to set on `element` now?

Reply:
Used the setupTests to refactor